Formula/j/jsontoolkit.rb

```diff
@@ -0,0 +1,98 @@
+class Jsontoolkit < Formula
+  desc "Swiss-army knife library for expressive JSON programming in modern C++"
+  homepage "https://jsontoolkit.sourcemeta.com/"
+  url "https://github.com/sourcemeta/jsontoolkit/archive/refs/tags/v2.0.0.tar.gz"
+  sha256 "00f82f02166beabec80522e2bbc7b839ee9b7ccb631411c42e6fab65186e80ba"
+  license "AGPL-3.0-only"
+
+  depends_on "cmake" => :build
+
+  def install
+    args = %W[
+      -DCMAKE_COMPILE_WARNING_AS_ERROR=OFF
+      -DJSONTOOLKIT_CONTRIB=OFF
+      -DJSONTOOLKIT_TESTS=OFF
+      -DJSONTOOLKIT_DOCS=OFF
+      -DBUILD_SHARED_LIBS=ON
+      -DCMAKE_INSTALL_RPATH=#{rpath}
+    ]
+
+    system "cmake", "-S", ".", "-B", "build", *args, *std_cmake_args
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+  end
+
+  test do
+    (testpath/"CMakeLists.txt").write <<~EOS
+      cmake_minimum_required(VERSION 3.15)
+      project(JsonToolkitTest)
+      set(CMAKE_CXX_STANDARD 20)
+
+      find_package(JSONToolkit REQUIRED)
+
+      add_executable(test test.cpp)
+      target_include_directories(test PRIVATE #{include})
```

Comment:
What is this include variable in Homebrew, out of curiosity? Why is it needed here?

Reply:
I tried to reuse: https://github.com/Homebrew/homebrew-core/pull/171151/commits/6e2458a4f0efbcf72ad04022eedca861ea081cf7#diff-174232a7f8493e9a0b394f9bb3221b2c9f6a411e02718c4f9254b0785ffb3dd3R41

Reply:
Ahhh makes sense